internal/module/openapi.go

```diff
@@ -0,0 +1,251 @@
+package module
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-openapi/spec"
+	"github.com/mohae/deepcopy"
+	"helm.sh/helm/v3/pkg/chartutil"
+
+	"github.com/deckhouse/d8-lint/internal/valuesvalidation"
+)
+
+const (
+	ExamplesKey = "x-examples"
+	ArrayObject = "array"
+	ObjectKey   = "object"
+)
+
+// applyDigests if ugly because values now are strongly untyped. We have to rewrite this after adding proper global schema
+func applyDigests(digests map[string]any, values any) {
+	if values == nil {
+		return
+	}
+	value, ok := values.(map[string]any)["global"]
```

Comment:
whats about cast one time?
```suggestion
        casted:=values.(map[string]any)
	value, ok :=casted["global"]
```